package.json

```diff
@@ -2,9 +2,12 @@
     "name": "@cartesi/rollups",
     "version": "1.2.0",
     "scripts": {
+        "build:contracts": "forge build contracts",
+        "build:devnet": "cannon build --write-deployments deployments --anvil.dump-state state.json",
```

Comment:
what about chain-id? does this build for 31337? Is the dump any different according to chain-id?

Reply:
Building it for chain ID 31337 is more complicated.